### ISSUE TITLE
Failed to run gmx make_ndx. Error code 11

### ISSUE THREAD

**Issue by camputchem** [luancarvalhomartins/PyAutoFEP#99]
I am running the tutorial and getting following error message. I am guessing the files are created on the fly in tmp directory and getting deleted as I can't find those files. Any suggestions? 

================== Align poses ===================
Molecule FXR_12 aligned
================ Working on pairs ================
Perturbation Pose Coordinates
============ Working on FXR_12-FXR_74 ============
FXR_12→FXR_74
================ Building system =================

=================== STACK INFO ===================
  File "/usr/local/PyAutoFEP/prepare_dual_topology.py", line 4241, in <module>
    build_data = prepare_complex_system(structure_file=arguments.structure,
  File "/usr/local/PyAutoFEP/os_util.py", line 549, in wrap_trace
    return f(*args, **kwargs)
  File "/usr/local/PyAutoFEP/prepare_dual_topology.py", line 703, in prepare_complex_system
    make_index(build_files_dict['index_ndx'], build_files_dict['fullsystem_pdb'], index_groups, selection_method,
  File "/usr/local/PyAutoFEP/prepare_dual_topology.py", line 1906, in make_index
    gmx_data = os_util.run_gmx(gmx_bin, index_cmd, input_data='{}\nq\n'.format(groups_str),
  File "/usr/local/PyAutoFEP/os_util.py", line 154, in run_gmx
    local_print('Failed to run {} {}. Error code {}.\nCommand line was: {}\n\nstdout:\n{}\n\nstderr:\n{}'
  File "/usr/local/PyAutoFEP/os_util.py", line 307, in local_print
    formatted_string = '\n{:=^50}\n{}{:=^50}\n'.format(' STACK INFO ', ''.join(traceback.format_stack()),
=================== STACK INFO ===================
[ERROR] Failed to run gmx make_ndx. Error code -11.
[ERROR] Command line was: ['gmx', 'make_ndx', '-n', '/tmp/tmpofcvryjn/tutorial/FXR_12-FXR_74/protein/build_system_160710_13072023/index.ndx', '-o', '/tmp/tmpofcvryjn/tutorial/FXR_12-FXR_74/protein/build_system_160710_13072023/index.ndx']
[ERROR]
[ERROR] stdout:
[ERROR]
[ERROR]
[ERROR] stderr:
[ERROR]                       :-) GROMACS - gmx make_ndx, 2023 (-:
[ERROR]
[ERROR] Executable:   /usr/local/gromacs/bin/gmx
[ERROR] Data prefix:  /usr/local/gromacs
[ERROR] Working dir:  /home/rajan/Desktop/fep_tut
[ERROR] Command line:
[ERROR]   gmx make_ndx -n /tmp/tmpofcvryjn/tutorial/FXR_12-FXR_74/protein/build_system_160710_13072023/index.ndx -o /tmp/tmpofcvryjn/tutorial/FXR_12-FXR_74/protein/build_system_160710_13072023/index.ndx
[ERROR]
[ERROR]

**Comment by xy21hb** [comment]
![image](https://github.com/luancarvalhomartins/PyAutoFEP/assets/14272986/641078e2-86f1-476a-a586-4cc5d9aeee34)

Change the Gromacs version to the designated.

**Comment by luancarvalhomartins** [comment]
Yes, GROMACS 2023 won't work, but the error should not come in the `make_ndx`. I suggest running with `output_hidden_temp_dir=False` so we can have the intermediate files to check.

Also note that this maybe a version of #6. In case, it should be fixed by using `output_hidden_temp_dir=False`.
